op.c

```diff
@@ -12041,12 +12041,33 @@ S_process_special_blocks(pTHX_ I32 floor, const char *const fullname,
             const I32 oldscope = PL_scopestack_ix;
             dSP;
             (void)CvGV(cv);
-            if (floor) LEAVE_SCOPE(floor);
-            ENTER;
-            PUSHSTACKi(PERLSI_REQUIRE);
-            SAVECOPFILE(&PL_compiling);
-            SAVECOPLINE(&PL_compiling);
+	    if (floor) LEAVE_SCOPE(floor);
```

Comment:
Something weird is going on indentation-wise here. It might be due to mixed tabs-vs-spaces. Make sure the new lines have only spaces, no tabs.